### PR TITLE
Set correct storybook canvas bg color

### PR DESCRIPTION
**Pull request by gsouquet**
Requires https://github.com/vector-im/compound-design-tokens/pull/45

Fixes the issue raised by @t3chguy in https://github.com/vector-im/compound-web/pull/20

- Set correct storybook canvas bg color
- Add theme selector dropdown

Screenshot 2023-04-26 at 14 18 56

### CODE REVIEW

**Comment by t3chguy** [comment]
The bg isn't changing for me

image

**Comment by gsouquet** [comment]
Yes, it requires vector-im/compound-design-tokens#45 to land first. Got tricked by CSS specificity unfortunately!

File: .storybook/preview.ts [hunk 1]
```diff
@@ -9,4 +9,22 @@ export const parameters = {
...
+      { name: "dark", class: "cpd-theme-dark", color: "#101317" },
+      {
+        name: "light-high-contrast",
+        class: "cpd-theme-light-hc",
+        color: "#fff",
+      },
+      {
+        name: "dark-high-contrast",
+        class: "cpd-theme-dark-hc",
+        color: "#101317",
```
Comment (alunturner) [inline .storybook/preview.ts:26]: Probably a silly question - how come we have to hardcode these hex values?

**Review by alunturner — APPROVED** [review]

File: .storybook/preview.ts (hunk 1 above)
Comment (gsouquet) [reply .storybook/preview.ts:26]: We don't have access to the compound design tokens there. Which is why we had to hardcode them

**Pull request merged**